wherescape/connectors/hubspot/collect_data.py

```diff
@@ -47,6 +54,9 @@ def hubspot_get_token(wherescape_instance: WhereScape, table_name: str):
     table_words = table_name.split("_")
 
     logging.info("retrieving access_token")
+    if develop_env:
+        logging.info("using developmental environment")
+        return parameter_name + "_sandbox"
```

Comment:
Does this work? was this tested? function is called `hubspot_get_token()`, however you're returning just a name not the token.